src/components/Modals/Onboarding.vue

```diff
@@ -192,13 +192,13 @@
 </template>
 
 <script>
-import reqsData from '@/requirements/reqs.json';
```

Comment:
Just for clarification, this won't change our workflow for data entry because it reads from reqs.json at requirement-sources right?

Reply:
It will. Now you will need to change it in `reqs.json`, and run the generator in `requirement-generator` folder.

Reply:
I see. So just for adding data, this won't change anything

Reply:
Yes

Reply:
Awesome. All good now